charts/kubeai/values.yaml

```diff
@@ -4,7 +4,12 @@
 
 secrets:
   huggingface:
+    create: true
```

Comment:
we should probably default to false, so it's not a hard requirement to supply a HF token? @nstogner your thought?

Reply:
With [the conditionals](https://github.com/substratusai/kubeai/pull/135/files#diff-88a0eb93bdcb1a4e615ab6e8751c47beaeadbbf57e9646ba071dd8bd418da6c7R69-R73) that I added, a HF token is a hard requirement.
Currently if you don't specify a token a secret is still created just with an empty value. When you go to pull a model that requires a token it will fail but you have to read the model's pod logs to figure out why.